operations/app/src/modules/network/main.tf

```diff
@@ -90,6 +90,32 @@ resource "azurerm_subnet_network_security_group_association" "private_private" {
   network_security_group_id = azurerm_network_security_group.nsg_private.id
 }
 
+resource "azurerm_virtual_network" "virtual_network_2" {
+  name = "${var.resource_prefix}-vnet2"
+  location = "westus"
+  resource_group_name = var.resource_group
+  address_space = ["11.0.0.0/16"]
+
+  tags = {
+    environment = var.environment
+  }
+}
+
+resource "azurerm_subnet" "private2" {
+  name = "private"
+  resource_group_name = var.resource_group
+  virtual_network_name = azurerm_virtual_network.virtual_network_2.name
+  address_prefixes = ["11.0.3.0/24"]
```

Comment:
The `11.` ip space is not reserved, so this will cause problems. The VPN PR is also going to use some of the `10.x` space, so to avoid collisions, I'm thinking set this to `10.0.23.0/24`, with the convention of `10.0.2x.0/24` being our peered subnets.

Reply:
Good catch.  I will make this update.